vm/devices/net/mana_driver/src/lib.rs

```diff
@@ -10,6 +10,5 @@ mod gdma_driver;
 pub mod mana;
 pub mod queues;
 mod resources;
```

Comment:
Can this crate become forbid(unsafe_code) now?

Reply:
It built with this added, so I believe it can

Reply:
Woo!